starter-code/lib/abbeyRoad.js

```diff
@@ -1,3 +1,5 @@
+var _ = require('lodash');
```

Comment:
GRACIAS sois de los poquitos que al volver de REPL habéis puesto lodash... GOOD JOB